features/login/impl/src/main/kotlin/io/element/android/features/login/impl/changeaccountprovider/form/ChangeAccountProviderFormView.kt

```diff
@@ -0,0 +1,218 @@
+import io.element.android.libraries.architecture.Async
+import io.element.android.libraries.designsystem.atomic.molecules.IconTitleSubtitleMolecule
+import io.element.android.libraries.designsystem.components.button.BackButton
+import io.element.android.libraries.designsystem.components.form.textFieldState
+import io.element.android.libraries.designsystem.preview.ElementPreviewDark
+import io.element.android.libraries.designsystem.preview.ElementPreviewLight
+import io.element.android.libraries.designsystem.theme.components.CircularProgressIndicator
+import io.element.android.libraries.designsystem.theme.components.Icon
+import io.element.android.libraries.designsystem.theme.components.IconButton
+import io.element.android.libraries.designsystem.theme.components.OutlinedTextField
+import io.element.android.libraries.designsystem.theme.components.Text
+import io.element.android.libraries.testtags.TestTags
+import io.element.android.libraries.testtags.testTag
+
+/**
+ * https://www.figma.com/file/o9p34zmiuEpZRyvZXJZAYL/FTUE?type=design&node-id=611-61435
+ */
+@Composable
+fun ChangeAccountProviderFormView(
+    state: ChangeAccountProviderFormState,
+    modifier: Modifier = Modifier,
+    onBackPressed: () -> Unit = {},
+    onLearnMoreClicked: () -> Unit = {},
+    onDone: () -> Unit = {},
+) {
+    val eventSink = state.eventSink
+    val scrollState = rememberScrollState()
+
+    Scaffold(
+        modifier = modifier,
+        topBar = {
+            TopAppBar(
+                title = {},
+                navigationIcon = { BackButton(onClick = onBackPressed) }
+            )
+        }
+    ) { padding ->
+        Box(
+            modifier = Modifier
+                .fillMaxSize()
+                .imePadding()
+                .padding(padding)
+                .consumeWindowInsets(padding)
+        ) {
+            Column(
+                modifier = Modifier
+                    .verticalScroll(
+                        state = scrollState,
+                    )
+            ) {
+                IconTitleSubtitleMolecule(
+                    modifier = Modifier.padding(top = 16.dp, bottom = 40.dp, start = 16.dp, end = 16.dp),
+                    iconImageVector = Icons.Filled.Home,
+                    title = stringResource(id = R.string.screen_account_provider_form_title),
+                    subTitle = stringResource(id = R.string.screen_account_provider_form_subtitle),
+                )
+
+                // TextInput
+                var userInputState by textFieldState(stateValue = state.userInput)
+
+                OutlinedTextField(
+                    value = userInputState,
+                    // readOnly = isLoading,
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(start = 16.dp, end = 16.dp, bottom = 30.dp)
+                        .testTag(TestTags.changeServerServer),
```

Comment:
It would be nice to have the `onTabOrEnterKeyFocusNext` modifier used here too, to improve the usage of this form in the emulator.